models/DataObject/Service.php

```diff
@@ -446,6 +417,16 @@ public static function gridObjectData($object, $fields = null, $requestedLanguag
                         }
                     }
 
+                    // because the key for the classification store has not a direct getter, you have to check separately if the data is inheritable
+                    if (strpos($key, '~') === 0 && empty($data[$key])) {
```

Comment:
guess the data-types isEmpty method should be used instead.

does (strpos($key, '~') === 0 also work for bricks and so on ?

Reply:
> guess the data-types isEmpty method should be used instead.

For Classification-Store keys `$def` is `null`. So `isEmpty` does not work.

> does (strpos($key, '~') === 0 also work for bricks and so on ?

Right, I'll fix that right now.

Reply:
Fixed in commit: 3e03840b05b74a5aa98a9440d891d492bb4307ad